submissions/vl-shevchenko/html-css-popup/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <link rel="icon" href="popup-icons2/favicon.png" type="images/png">
+        <title>Hooli-style Popup</title>
+    </head>
+    <body>
+        <header>
+            <nav>
+                <div>                                 
+                    <div class="conteiner_menu">                   
+                        <ul class="conteiner__unlist">
+                            <li class="list"><a href="#">Mail</a></li>
+                            <li class="list"><a href="#">Images</a></li>
+                            <li>
+                                <input tabindex="-1" class="checkbox_knopka" type="checkbox" id="checkbox_knopka">
+                                <label class="list" for="checkbox_knopka">                               
+                                    <a tabindex="0"><img src="popup-icons2/01.png" alt=""></a>
+                                </label>         
+                                <div class="conteiner__list-hiden">                       
+                                    <div class="conteiner">
+                                        <ul class="unlist-list-hiden">
+                                            <li class="list-hiden"><a href="#"><img src="popup-icons2/calendar.png" alt="">Calendar</a></li>
+                                            <li class="list-hiden"><a href="#"><img src="popup-icons2/news.png" alt="">News</a></li>
+                                            <li class="list-hiden"><a href="#"><img src="popup-icons2/maps.png" alt="">Map</a></li>
+                                            <li class="list-hiden"><a href="#"><img src="popup-icons2/files.png" alt="">Files</a></li>
+                                            <li class="list-hiden"><a href="#"><img src="popup-icons2/contacts.png" alt="">Contacts</a></li>
+                                            <li class="list-hiden"><a href="#"><img src="popup-icons2/videos.png" alt="">Video</a></li>
+                                            <li class="list-hiden"><a href="#"><img src="popup-icons2/apps.png" alt="">Apps</a></li>
+                                            <li class="list-hiden"><a href="#"><img src="popup-icons2/promo.png" alt="">Promo</a></li>
+                                        </ul>
+                                        <input tabindex="-1" type="checkbox" id="checkbox_more">
+                                        <label tabindex="0" class="checkbox_more-btn more-btn" for="checkbox_more" >
+                                            <a class="more-btn">More</a>
+                                        </label>                         
+                                        <hr class="more-hr">
```

Comment:
Avoid using HTML elements for decorative purposes use CSS for it(pseudoelements, borders, etc)